frontend/client-list/client-list.component.tsx

```diff
@@ -18,9 +18,7 @@ export default function ClientList(props: ClientListProps) {
   useClientsApi(apiState, dispatchApiState);
   useFrontendUrlParams(apiState, dispatchApiState);
 
-  const fetchingClient =
-    apiState.status === ApiStateStatus.fetching ||
-    apiState.status === ApiStateStatus.shouldFetch;
+  const fetchingClient = apiState.status !== ApiStateStatus.fetched;
```

Comment:
The bug was caused by `fetchingClient` starting out as `false` because the initial state was starting with the apiState.status being `fetched`.